scipy/optimize/_highs/cython/src/_highs_wrapper.pyx

```diff
@@ -112,7 +112,7 @@ cdef str _opt_warning(string name, val, valid_set=None) noexcept:
            'See documentation for valid options. '
            'Using default.' % (name.decode(), str(val)))
 
-cdef apply_options(dict options, Highs & highs) noexcept:
+cdef void apply_options(dict options, Highs & highs) noexcept:
```

Comment:
This `noexcept` might be useful, however it is ignored, adding a return type means the `noexcept` is no longer ignored.